addons/repair/i18n/repair.pot

```diff
@@ -358,7 +360,7 @@ msgstr ""
 
 #. module: repair
 #: model_terms:ir.ui.view,arch_db:repair.stock_warn_insufficient_qty_repair_form_view
-msgid "Do you confirm you want to repair"
+msgid "<br/>You can still confirm the repair of"
```

Comment:
Looks like you got your text split in two; that makes it hard to translate. Not that it was any better before your changes, though.